toolchain/autoupdate_testdata.py

```diff
@@ -8,17 +8,33 @@
 SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 """
 
+import re
 import subprocess
 import sys
 from pathlib import Path
 
 
 def main() -> None:
+    # Use the most recently used build mode, or `fastbuild` if missing
+    # `bazel-bin`.
+    build_mode = "fastbuild"
+    workspace = subprocess.check_output(
+        ["bazel", "info", "workspace"], encoding="utf-8"
+    ).strip()
```

Comment:
For me this produces two lines of output (an `INFO: Invocation ID: ...` line and a path). Do we need:
```suggestion
    workspace = subprocess.check_output(
        ["bazel", "info", "workspace", "--ui_event_filters=-info"], encoding="utf-8"
    ).strip()
```
to turn off the `INFO` logging? (Maybe this is working for you because you have that in your `.bazelrc`?)

Reply:
Looks like the INFO line goes to stderr, but it's still extra noise that we generally turn off in this script.

Reply:
It works because this is reading stdout, and that's on stderr. Is it important to disable? (I'm hesitant to turn off output in case there's an issue)

Reply:
So far in this script we've been turning off extra noise like this -- see the `ui_event_filters` flag below.

Reply:
Oh, I hadn't even noticed that. Not really a fan of turning this stuff off, but adding for consistency.